### PR TITLE
Update CLI dependencies for v1-lts

### PR DESCRIPTION
**Pull request by t1m0thyj**
## Release Notes

Milestone: 1.22.8

Changelog: Update dependencies for technical currency purposes

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/zowe-explorer-vscode/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [ ] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

**Review by traeok — DISMISSED** [review]
LGTM, thanks Timothy

**Review by zFernand0 — DISMISSED** [review]

File: packages/zowe-explorer/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### Bug fixes
 
+- Update dependencies for technical currency purposes.
```
Comment (anaxceron) [inline packages/zowe-explorer/CHANGELOG.md:9]: Can we add an issue or pr link to this?

**Review by anaxceron — APPROVED** [review]
Left one small comment, otherwise this is good to go as far as I'm concerned @t1m0thyj.

**Review by traeok — APPROVED** [review]

**Review by anaxceron — DISMISSED** [review]
👍

**Review by zFernand0 — APPROVED** [review]

**Review by JillieBeanSim — APPROVED** [review]
Thanks @t1m0thyj

**Review by zFernand0 — CHANGES_REQUESTED** [review]
Should we combine this with:
- https://github.com/zowe/zowe-cli/pull/2172

**Review by zFernand0 — APPROVED** [review]
Thanks for updating the CLI version (again) 😋 

LGTM! 😋 

I believe this is be good to be merged ASAP since the Zowe V1 release is now live 😋

**Review by JillieBeanSim — APPROVED** [review]

**Review by traeok — APPROVED** [review]

**Pull request merged**